scripts/west_commands/run_common.py

```diff
@@ -568,6 +567,12 @@ def load_cmake_cache(build_dir, args):
         log.die(f'no CMake cache found (expected one at {cache_file})')
 
 def rebuild(command, build_dir, args):
+    skip_rebuild_config = config.get('flash', 'skip-rebuild', fallback='false')
```

Comment:
I'm not against this proposal, but currently the `flash.skip-rebuild` option would also be used for `debug`/`simulate`/`robot`, which feels wrong.

It can be "fixed" with
```suggestion
    skip_rebuild_config = config.get(command.name, 'skip-rebuild', fallback='false')
```

But this also can become somewhat of a footgun.

An alternative (already existing) solution is to create an alias (can be set globally too):

```shell
$ west config alias.flash -- flash --skip-rebuild
```